src/classes/ALLO_PaymentSync_TEST.cls

```diff
@@ -618,6 +618,213 @@ private with sharing  class ALLO_PaymentSync_TEST {
             'Inserting or updating opportunity and payment allocations for the same opportunity failed to cause the expected exception.');
```

Comment:
This is not part of this change, but do you mind moving this to a label?

Reply:
@jbrandolinisf Should this be moved into a variable instead of a label? I don't think this is necessary as a label since this is for test class use only.

Reply:
Oh, right, never mind.  Leave it as is.